### PR TITLE
Fix to compile in Swift4

### PR DESCRIPTION
**Pull request by sirioz**

### CODE REVIEW

File: Typist.swift [hunk 1]
```diff
@@ -103,11 +103,12 @@ public class Typist: NSObject {
     
     /// Stops listening to keyboard events. Callback closures won't be cleared, thus calling `start()` again will resume calling previously set event handlers.
     public func stop() {
+        // TODO: this needs a bit more work
```
Comment (totocaster) [inline Typist.swift:106]: Can you elaborate on "more work" in comment itself? Thanks.

**Review by totocaster — COMMENTED** [review]
Thanks for the PR! let's discuss few points and I'll merge it with the master.

File: Typist.swift (hunk 1 above)
Comment (sirioz) [reply Typist.swift:106]: I don't know what happened, I did not write it. It was already there when I did my changes.

Comment (totocaster) [reply Typist.swift:106]: Ah, it seems I've removed it in time you had pulled code and submitted the PR. OK, all clear. I'll merge and release this weekend. Thanks!

File: Typist.swift [hunk 2]
```diff
@@ -82,7 +82,7 @@ public class Typist: NSObject {
         case didChangeFrame
     }
     
-    /// Declares Typist behavior. Pass a closure parameter and event to bind those two. Without calling `start()` none of the closures will be executed.
```
Comment (totocaster) [inline Typist.swift:85]: This is not a typo. Bind, meaning to connect two (event <-> action).

**Comment by totocaster** [comment]
Merged, I'll fix the rest and release. Thanks!

**Comment by totocaster** [comment]
Released as 1.1.0 🚀

**Comment by cgossain** [comment]
@totocaster Thanks for this. Could you push a new podspec to the cocoapods trunk? Thanks!

**Pull request merged**